routes/v1.js

```diff
@@ -766,6 +772,7 @@ api.declare({
   method:     'post',
   route:      '/task/:taskId/rerun',
   name:       'rerunTask',
+  stability:  base.API.stability.experimental,
```

Comment:
multiple pulse messages are saying particular runs have been resolved in different ways...same way if we hit infra issues that cause a rerun.